helm-chart/openj9-jitserver-chart/README.md

```diff
@@ -0,0 +1,228 @@
+helm repo update
+helm search repo openj9-jitserver-chart
+```
+
+Install OpenJ9 JITServer helm chart by following below command.
+``` bash
+helm install jitserver-release openj9-jitserver-chart
+```
+
+This chart deploys a JITServer instance with latest OpenJ9 Java 8 release on x86 platform by default. If your application uses a different version of Java, please include the option `--set image.tag="REPLACE_IMAGE_TAG"` in above command which overrides below section in `Values.yaml`. 
+```yaml
+image:
+  tag: 8-openj9
+```
+
+Java versions 8 and 11 are currently supported on x86, POWER and Z platforms.
+
+### 2. Enable your application to use JITServer.
+Once the JITServer is deployed, you can enable your application to use JITServer by adding the following JVM option as an environment variable into your Java application container.
+
+Service names can be found by `kubectl get service`, `<serverport>` is set to `38400` by default. 
+``` bash
+JAVA_OPTIONS = "-XX:+UseJITServer -XX:JITServerAddress=<servicename> -XX:JITServerPort=<serverport>"
+```
+
+`<servicename>` is set automatically by helm, and you may change `<serverport>` by including option `--set service.port="REPLACE_PORT_NUMBER"`. This option overrides below section in `Values.yaml`. 
+```yaml
+service:
+  port: 38400
+```
+
+Setting the `JAVA_OPTIONS` environment variable on a new or an already in production application depends on your application build/deployment pipeline. For more detailed instructions on how to redeploy your application with the above environment variable changes, please read through [this tutorial](./enable-jitserver.md). The tutorial shows how to enable JITServer technology for an example OpenLiberty Java application.
+
+## JITServer considerations and platform support
+### How to select a platform and a Java version 
+Platforms are handled by multi-arch images from AdoptOpenJDK docker hub, thus platform specific JITServer is deployed based on which platform the install is being done on. The JITServer Java version has to match the application Java version. For example, Java 8 applications need Java 8 JITServer.
+
+If your application uses a different version of Java, please include the option `--set image.tag="REPLACE_IMAGE_TAG"` in the `helm install` command which overrides below section in `Values.yaml`.
+
+Alternatively, if you have a copy of the helm chart then, in order to select a specific java version, you can edit `image.repository` and `image.tag` fields inside `Values.yaml` to change the image location.
+
+For example, this helm chart pulls `adoptopenjdk:8-openj9` image and runs `jitserver`.
+``` yaml
+image:
+  repository: adoptopenjdk
+  tag: 8-openj9
+  pullPolicy: Always
+  command: ["jitserver"]
+```
+
+Should you want to deploy AdoptOpenJDK Java 11 images, change `image` fields inside `Values.yaml` as shown below.
+``` yaml
+image:
+  repository: adoptopenjdk
+  tag: 11-openj9
+  pullPolicy: Always
+  command: ["jitserver"]
+```
```

Comment:
This is the equivalent of using the `latest` tag which I understand is not a good practice.  Can you update this to specify a specific image tag?

Reply:
@dinogun any suggestions on how to define this?

Reply:
@chrisc66 Any update on using a release-specific tag here as the example?  Pulling "latest" is an anti-practice as it breaks repeatable deploys as "latest" changes outside the deployers control

Reply:
@chrisc66 I'm just looking through the comments before hitting approve and see this was never responded to.  Do you have an update here?

Reply:
Found below image tags from AdoptOpenJDK official docker repo that point to the 0.24 release image and support multiple architectures. 
- [`adoptopenjdk:11.0.10_9-jdk-openj9-0.24.0`](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/11.0.10_9-jdk-openj9-0.24.0/images/sha256-48ccf272c68adf96eb83794b53b5539124762c1ceb932673e5dafb0fbc426295?context=explore)
- [`adoptopenjdk:8u282-b08-jdk-openj9-0.24.0`](https://hub.docker.com/layers/adoptopenjdk/library/adoptopenjdk/8u282-b08-jdk-openj9-0.24.0/images/sha256-e5f59bd1041890389b5d581814f5d0bf00452dfb575082557326e6d1a6f3c74e?context=explore)